doc/en/components/grids/_shared/toolbar.md

```diff
@@ -0,0 +1,669 @@
+
+The following code snippet demonstrates subscribing to the toolbar exporting event and configuring the exporter options:
+
+```html
+<@@igSelector (toolbarExporting)="configureExport($event)" ></@@igSelector>
+```
+
+```typescript
+configureExport(args: IGridToolbarExportEventArgs) {
+    const options: IgxExporterOptionsBase = args.options;
+
+    options.fileName = `Report_${new Date().toDateString()}`;
+
+    if (options instanceof IgxExcelExporterOptions) {
+        options.columnWidth = 10;
+    } else {
+        options.fileType = CsvFileTypes.TSV;
+        options.valueDelimiter = '\t';
+    }
+
+    args.exporter.columnExporting.subscribe((columnArgs: IColumnExportingEventArgs) => {
+        @@if (igxName === 'IgxGrid') {
+        // Don't export image fields
+        columnArgs.cancel = columnArgs.header === 'Athlete' ||
+                            columnArgs.header === 'Country';
+        }
+        @@if (igxName === 'IgxTreeGrid') {
+        // Don't export image field
+        columnArgs.cancel = columnArgs.header === 'Name';
+        }
+    });
+}
+```
+}
+
+The following sample demonstrates how to customize the exported files:
+
+@@if (igxName === 'IgxGrid') {
+
+<code-view style="height:420px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/grid/grid-toolbar-sample-3" >
+</code-view>
+
+<div class="divider"></div>
+}
+@@if (igxName === 'IgxTreeGrid') {
+
+<code-view style="height:420px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/tree-grid/treegrid-toolbar-2" >
+</code-view>
+
+<div class="divider"></div>
+}
+
+## Exporting Indicator
+
+When using the default toolbar exporter component, whenever an export operation takes place the toolbar will show a progress indicator while the operation is in progress.
+Moreover, users can set the toolbar [showProgress]({environment:{Platform}ApiUrl}/classes/igxgridtoolbarcomponent.html#showprogress) property and use for their own long running operations
+or just as another way to signify an action taking place in the grid.
+The sample belows uses has significant amount of data, in order to increase the time needed for data export so the progressbar can be seen. Additionally it has another button
+that simulates a long running operation in the grid:
+
+@@if (igxName === 'IgxGrid') {
+
+<code-view style="height: 370px;" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/grid/exporting-visualization" >
+</code-view>
+
+<div class="divider--half"></div>
+}
+@@if (igxName === 'IgxTreeGrid') {
+
+<code-view style="height: 370px;" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/tree-grid/tree-grid-exporting-visualization" >
+</code-view>
+
+<div class="divider--half"></div>
+}
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.